package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@axa-ch/patterns-library",
+  "version": "3.0.0",
```

Comment:
Watch out, this declares a stable `3.0.0` release according to semantic versioning.
Guess what you need is `next`, `alpha`, `beta` etc. tag.

More details at: https://semver.org/